### ISSUE TITLE
TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType' in Schedule.Validate

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#106]
```
What steps will reproduce the problem?
1. Run the thing.

What is the expected output? What do you see instead?
I expect to see whether or not my data validates as GTFS. I see an error
instead.

What version of the product are you using? On what operating system?
v. 1.1.7 on Ubuntu Linux 8.10 x86_64 with Python 2.5.2

Please provide any additional information below.
There was a crash report, so I sent it along.

```

Original issue reported on code.google.com by `plattyp...@gmail.com` on 2 Nov 2008 at 8:21

Attachments:
- [validation-crash.txt](https://storage.googleapis.com/google-code-attachments/googletransitdatafeed/issue-106/comment-0/validation-crash.txt)

**Comment by GoogleCodeExporter** [comment]
```
looks like there is a problem with stop_lat and stop_lon in stops.txt; the stop
objects in the crash report don't have these attributes. Please send the first 
few
lines of your file so I can add a test.

The change in http://codereview.appspot.com/7889/show should prevent the crash. 
You
can apply it by changing to the directory containing transitfeed.py and running
curl 'http://codereview.appspot.com/download/issue7889_1.diff' | patch -p0
or wait until the next release.
```

Original comment by `tom.brow...@gmail.com` on 2 Nov 2008 at 1:20
- Changed title: **TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType' in Schedule.Validate**
- Changed state: **Accepted**
- Added labels: **App-FeedValidator**, **Language-Python**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.1.8
```

Original comment by `tom.brow...@gmail.com` on 3 Dec 2008 at 8:54
- Changed state: **Fixed**
